src/layout/header/MenuButton.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, ButtonProps } from '@mui/material';
+import { Link, LinkProps } from 'react-router-dom';
+
+interface MenuButtonProps extends ButtonProps, Pick<LinkProps, 'to'> {
+  isSelected?: boolean;
```

Comment:
Det er kanskje ikke noen grunn til å tillate undefined her? 
```suggestion
  isSelected: boolean;
```